runtimes/standalone/src/lib.rs

```diff
@@ -666,8 +647,7 @@ construct_runtime!(
 		// // System scheduler.
 		// Scheduler: pallet_scheduler = 32,
 
-		// Vesting. Usable initially, but removed once all vesting is finished.
-		Vesting: pallet_vesting = 33,
```

Comment:
Did you check with BTE whether removing Vesting does not introduce a critical change? AFAIK, we still have vesting in Spiritnet and thus they will probably have some unit tests involving vesting?

Reply:
But BTE would never use the mashnet-node dev, right? They would use Peregrine, which still has the vesting stuff.

Reply:
The vesting stuff was not populated in the genesis. Plus, we are talking about a dev environment. I will ask the BTE people whether they need it. If they don't for the near future, I would suggest to leave it as it is an re-enable it later on if and when needed.